inference-engine/src/vpu/graph_transformer/src/stages/exp_generateproposals.cpp

```diff
@@ -129,6 +131,13 @@ void FrontEnd::parseExpGenerateProposals(
         outputs);
 
     stage->attrs().set("params", params);
+
+    const int numProposals = sizeof(uint8_t) * 2 * inputScores->desc().dim(Dim::H) * inputScores->desc().dim(Dim::W) * inputScores->desc().dim(Dim::C);
```

Comment:
`sizeof(uint8_t) = 1`. 
What does this "magic" operation `sizeof(uint8_t) * 2` mean?
Please, use named constants for such purposes

Reply:
The sizeof is used at the detection_output. Change to constants anyway?